packages/deployments/contracts/contracts/core/connext/facets/BridgeFacet.sol

```diff
@@ -495,6 +494,19 @@ contract BridgeFacet is BaseConnextFacet {
         // Set boolean flag
         isCanonical = _params.originDomain == canonical.domain && local == _asset;
 
+        // Enforce liquidity caps
+        // NOTE: safe to do this before the swap because canonical domains do
+        // not hit the AMMs (local == canonical)
+        if (isCanonical) {
+          // NOTE: this method includes router liquidity as part of the caps,
+          // not only the minted amount
+          uint256 custodied = IERC20(local).balanceOf(address(this)) + _amount;
```

Comment:
idk how i feel about the fact that the custodied funds are cumulative regardless of the toke